spec.bs

```diff
@@ -877,6 +877,7 @@ A [=tokenizer=] has an associated <dfn for=tokenizer>code point</dfn>, a Unicode
   1. Set |tokenizer|'s [=tokenizer/input=] to |input|.
   1. Set |tokenizer|'s [=tokenizer/policy=] to |policy|.
   1. While |tokenizer|'s [=tokenizer/index=] is less than |tokenizer|'s [=tokenizer/input=]'s [=string/code point length=]:
+    1. Set |tokenizer|'s [=tokenizer/next index=] to |tokenizer|'s [=tokenizer/index=].
```

Comment:
Can you instead combine these two lines into a single invocation of "seek and get the next code point"?  Something like:

`1. Run [=seek and get the next code point=] given |tokenizer| and |tokenizer|'s [=tokenizer/index=].`